content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -299,9 +299,9 @@ You can use the following environment variables to configure injection and extra
 - `DD_TRACE_PROPAGATION_STYLE_INJECT=Datadog, b3multi, tracecontext`
 - `DD_TRACE_PROPAGATION_STYLE_EXTRACT=Datadog, b3multi, tracecontext`
 
-The environment variable values are comma-separated lists of header styles enabled for injection or extraction. By default, only the `Datadog` injection style is enabled.
+The environment variable values are comma-separated lists of header styles enabled for injection or extraction. If multiple extraction styles are enabled, the extraction attempt is completed in the order of configured styles, and uses the first successful extracted value.
```

Comment:
While you're here, in the list of styles above, perhaps we could move the tracecontext to the top of the list that way it highlights that we're placing the most importance on that one? And in the example above, we could move tracecontext to the front so it looks like the default style + b3multi?